webui/react/src/pages/TrialDetails/MultiTrialDetailsHyperparameters.tsx

```diff
@@ -76,8 +76,10 @@ const MultiTrialDetailsHyperparameters: React.FC<Props> = ({
     ],
   );
 
+  const experimentIds = useMemo(() => [experiment.id], [experiment.id]);
```

Comment:
wrapped `[experiment.id]` with `useMemo` since raw object(array) cause re-rendering since it returns `false` in `Object.is()`